app/controllers/v0/sign_in_controller.rb

```diff
@@ -347,6 +345,24 @@ def create_login_code(state_payload, user_info, credential_level) # rubocop:disa
              content_type: 'text/html'
     end
 
+    def create_user_code_map(user_attributes, state_payload, verified_icn, request_ip)
+      if state_payload.client_id == Settings.sign_in.arp_client_id
+        AccreditedRepresentativePortal::RepUserCreator.new(
+          user_attributes:,
+          state_payload:,
+          verified_icn:,
+          request_ip:
+        ).perform
+      else
+        SignIn::UserCreator.new(
+          user_attributes:,
+          state_payload:,
+          verified_icn:,
+          request_ip:
+        ).perform
+      end
```

Comment:
You could probably condense this to something like:

```ruby
klass = case state_payload.client_id
        when Settings.sign_in.arp_client_id
          AccreditedRepresentativePortal::RepUserCreator
        else
          SignIn::UserCreator
        end

klass.new(user_attributes:, state_payload:, verified_icn:, request_ip:).perform
```

Reply:
I like Dick's suggestion. I was originally thinking of a ternary operator but we probably have future non-standard User types coming, which this handles well.

Reply:
Here's my alternate suggestion: lets just get rid of UserCreator (I wouldn't do it in Riley's PR, but in a follow up PR)

Reply:
Yeah I wasn't worrying about this too much cause it's just going to be removed. We could technically just leave it how it was with just `SignIn::UserCreator` and it would still work because current_user just gets overridden by the UserLoader's